### PR TITLE
scripts: update transaction TTL related metrics for  tidb grafana

### PR DESCRIPTION
**Pull request by tiancaiamao**
See http://172.16.5.34:34125/d/000000011/test-cluster-tidb?orgId=1

### CODE REVIEW

**Comment by tiancaiamao** [comment]
PTAL @jackysp @liubo0127

**Comment by liubo0127** [comment]
LGTM

File: scripts/tidb.json [hunk 1]
```diff
@@ -3319,1190 +3319,1389 @@
       "type": "row"
     },
     {
-      "collapsed": true,
+      "collapsed": false,
```
Comment (jackysp) [inline scripts/tidb.json]: It is better to make the panel collapsed, so keep it `true`.

File: scripts/tidb.json [hunk 2]
```diff
@@ -11088,16 +11287,16 @@
       "type": "row"
     }
   ],
-  "refresh": "30s",
+  "refresh": false,
```
Comment (jackysp) [inline scripts/tidb.json]: Why stop to refresh the panels?

File: scripts/tidb.json [hunk 3]
```diff
@@ -11088,16 +11287,16 @@
...
   "schemaVersion": 18,
   "style": "dark",
   "tags": [],
   "templating": {
     "list": []
   },
   "time": {
-    "from": "now-1h",
-    "to": "now"
+    "from": "2020-01-02T16:10:11.043Z",
```
Comment (jackysp) [inline scripts/tidb.json]: Why change this?

Comment (tiancaiamao) [reply scripts/tidb.json]: I find that if I do some operation and click export, the content of the json file would be different.
This time it should be correct.

Comment (jackysp) [reply scripts/tidb.json]: OK

**Pull request merged**